reco_analysis/reco_analysis/app/credentials.yaml

```diff
@@ -0,0 +1,36 @@
+credentials:
+  usernames:
+    "12305811":
```

Comment:
is the patient ID related to MIMIC ID?

Reply:
This is the patient ID corresponding to the transcripts that Annie produced.